docs/ref/models/fields.txt

```diff
@@ -502,6 +502,15 @@ Any combination of these options will result in an error.
     ``True`` will cause the field to have ``editable=False`` and ``blank=True``
     set.
 
+.. note::
+    When using timezones, storing a ``DateField`` in the database is
+    ambiguous because it's incapable of storing timezone aware information, unlike ``DateTimeField``.
```

Comment:
I don't think that's a good explanation. Dates aren't ambiguous, even in the presence of timezones.

Only when converting between datetimes and dates do ambiguities appear.

A date is a completely different concept from a datetime, not just a less accurate representation.